src/winetricks

```diff
@@ -10142,6 +10142,103 @@ load_ie8_kb2936068()
+        w_try "$WINE" iexplore -unregserver
+    fi
+
+    # FIXME: Do we need more? Or less....
+    # Change the override to the native so we are sure we use and register them
+    w_override_dlls native,builtin ieframe ieproxy jscript mshtml shdocvw shlwapi urlmon vbscript wininet xmllite
+
+    # Bundled updspapi cannot work on Wine
+    w_override_dlls builtin updspapi
+
+    #    Can we also use sed to replace iexplore.exe's version string???? No idea
+    #    Removing the fake iexplore.exe's doesn`t work. Installer still complains a newer version
+    #    of IE is installed and refuses to run. Replacing fake iexplore.exe with one that
+    #    has a lower version string works.
+    #    Fake iexplore.exe generated with https://bugs.winehq.org/attachment.cgi?id=65708&action=diff
+    #
+    #    Get fake iexplore with low version string and copy it to approriate places
+
+    w_download_to "$W_CACHE/$W_PACKAGE" https://bugs.winehq.org/attachment.cgi?id=65709 318ca190ffcc788e822fd7e0616ce44d9388c08261a3f1c19517e65a4f5c0732 "iexplore32.exe"  #32-bit
+    w_download_to "$W_CACHE/$W_PACKAGE" https://bugs.winehq.org/attachment.cgi?id=65710 65eaae7b2165fe4415bfb123eb179cbdd24d77ce41fe62c9ea5850f1ca83d0b6 "iexplore64.exe"  #64-bit
+
+    w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore64.exe" "$W_SYSTEM64_DLLS"/iexplore.exe
+    w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore32.exe" "$W_SYSTEM32_DLLS"/iexplore.exe
+    
+    w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore64.exe" "$WINEPREFIX/drive_c/Program Files/Internet Explorer/iexplore.exe"
+    w_try cp -f "$W_CACHE/$W_PACKAGE/iexplore32.exe" "$WINEPREFIX/drive_c/Program Files (x86)/Internet Explorer/iexplore.exe"
+
+    w_download "http://download.microsoft.com/download/7/5/4/754D6601-662D-4E39-9788-6F90D8E5C097/$file1" bcff753e92ceabf31cfefaa6def146335c7cb27a50b95cd4f4658a0c3326f499
+    if [ -n "$W_UNATTENDED_SLASH_QUIET" ]; then
+        quiet="$W_UNATTENDED_SLASH_QUIET /forcerestart"
+    fi
+    
+    #For some reason installer returns exit code 194 so no w_try 
+    "$WINE" "$W_CACHE/$W_PACKAGE/$file1" $quiet
+
+    # Builtin ieproxy is in system32, but ie8's lives in Program Files. Native
+    # CLSID path will get overwritten on prefix update. Setting ieproxy to
+    # native doesn't help because setupapi ignores DLL overrides. To work
+    # around this problem, copy native ieproxy to system32.
+    w_try cp -f "$WINEPREFIX/drive_c/Program Files (x86)/Internet Explorer/ieproxy.dll" "$W_SYSTEM32_DLLS"/ieproxy.dll
+    w_try cp -f "$WINEPREFIX/drive_c/Program Files/Internet Explorer/ieproxy.dll" "$W_SYSTEM64_DLLS"/ieproxy.dll
+
+    # Work around DLL registration bug until ierunonce/RunOnce/wineboot is fixed
+    for i in browseui.dll corpol.dll dxtmsft.dll dxtrans.dll ieaksie.dll ieapfltr.dll \
+             iedkcs32.dll iepeers.dll ieproxy.dll jscript.dll licmgr10.dll \
+             msdbg2.dll mshtmled.dll mstime.dll shdocvw.dll tdc.ocx urlmon.dll vbscript.dll
+    do
```

Comment:
Not your fault; but I recently made the style for do more consistent (which travis will enforce), please move it to the above line (`vbscript.dll; do)